### PR TITLE
The request time in milliseconds should not derived from its time in nanoseconds

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: test/uk/gov/hmrc/apigateway/service/EndpointServiceSpec.scala [hunk 1]
```diff
@@ -97,11 +99,13 @@ class EndpointServiceSpec extends UnitSpec with MockitoSugar {
 
       mockApiServiceConnectorToReturn("api-context", successful(anApiDefinition))
 
-      val beforeTime = System.nanoTime()
+      val beforeTimeMillis = System.currentTimeMillis()
+      val beforeTimeNanos = System.nanoTime()
       val actualApiRequest = await(endpointService.apiRequest(request))
-      val afterTime = System.nanoTime()
+      val afterTimeMillis = System.currentTimeMillis()
+      val afterTimeNanos = System.nanoTime()
```
Comment (tom59) [inline test/uk/gov/hmrc/apigateway/service/EndpointServiceSpec.scala]: I think using joda for time in millis is cleaner.
Unfortunately there is no joda for time in nanos but hopefully we will get rid of the time in nanos soon

Comment (gokyo) [inline test/uk/gov/hmrc/apigateway/service/EndpointServiceSpec.scala]: > I think using joda for time in millis is cleaner.
What do you mean @tom59 ?

http://stackoverflow.com/questions/17080643/jodatime-vs-system-currenttimemillis-in-terms-of-accuracy

Comment (tom59) [inline test/uk/gov/hmrc/apigateway/service/EndpointServiceSpec.scala]: I mean it is more testable as you can then use DateTimeUtils.setCurrentMillisFixed()

Comment (gokyo) [inline test/uk/gov/hmrc/apigateway/service/EndpointServiceSpec.scala]: got it

**Pull request merged**